### PR TITLE
Scannet doc

### PR DESCRIPTION
**Pull request by hjin2902**
Add docs of ScanNet including:

- data preparation: explanation about raw data and annotations
- train pipeline: load data and pre-processing
- metrics: evaluation

### CODE REVIEW

File: docs/datasets/scannet.md [hunk 1]
```diff
@@ -0,0 +1,403 @@
+# ScanNet
+
+## Dataset preparation
+
+For the overall process, please refer to [scannet](https://github.com/open-mmlab/mmdetection3d/blob/master/data/scannet/README.md/).
```
Comment (Tai-Wang) [inline docs/datasets/scannet.md:5]: refer to the \[README\]\(https://github.com/open-mmlab/mmdetection3d/blob/master/data/scannet/README.md/) page for ScanNet

File: docs/datasets/scannet.md [hunk 2]
```diff
@@ -0,0 +1,403 @@
...
+
+### Export ScanNet data
+
```
Comment (Tai-Wang) [inline docs/datasets/scannet.md:8]: Add a sentence to summarize what we do by exporting ScanNet data

**Review by Tai-Wang — COMMENTED** [review]
Maybe we can simplify the code scripts and replace them with your summary for the data processing steps. Too many codes will make the doc unnecessarily too long.

File: docs/datasets/scannet.md [hunk 3]
```diff
@@ -0,0 +1,351 @@
...
+    return mesh_vertices, label_ids, instance_ids, unaligned_bboxes, \
+        aligned_bboxes, object_id_to_label_id, axis_align_matrix
+
+```
+
+After exporting each scan, the raw point cloud could be downsampled, e.g. to 50000. In addition, invalid semantic label outside of `nyu40id` standard should be filtered. Finally, the point cloud data, semantic labels, instance labels and ground truth bounding boxes should be saved in `.npy` files. The overall `export_one_scan` procedure is as follows:
+
+```python
+def export_one_scan(scan_name,
```
Comment (ZwwWayne) [inline docs/datasets/scannet.md:143]: can be simplified.

**Review by ZwwWayne — APPROVED** [review]

**Pull request merged**